src/rust/engine/Cargo.lock

```diff
@@ -3092,7 +3092,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0c1318b19085f08681016926435853bbf7858f9c082d0999b80550ff5d9abe15"
 dependencies = [
  "bytes",
- "heck 0.5.0",
+ "heck 0.4.0",
```

Comment:
Do you know why this was downgraded? Seems a bit surprising

Reply:
I don't know. The change was made by running `cargo update -p pyo3`. I re-verified this by re-running that command and the change still occurs.

Reply:
The change was made to the stanza for `prost-build`. See https://crates.io/crates/prost-build/0.13.3/dependencies. It depends on `heck` with `>=0.4, <=0.5` (instead of `<0.5`). So `cargo` thought to change the transitive dependency for some reason since the change still matches the constraint. Maybe an edge case in `cargo` constraint solver?

Reply:
Similar issue reported to cargo: https://github.com/rust-lang/cargo/issues/14446

Reply:
I "fixed" the problem by running `cargo update -p heck@0.4.0` and it moved `heck` to v0.4.1 and magically put the `prost-build` dependency back to `heck` v0.5.0.